packages/chaire-lib-backend/src/utils/processManagers/osrmProfiles/walking_no_penalty.lua

```diff
@@ -115,28 +115,28 @@ function setup()
 
     speeds = Sequence {
       highway = {
-        trunk           = walking_speed*0.98,
-        trunk_link      = walking_speed*0.98,
-        primary         = walking_speed*0.98,
-        primary_link    = walking_speed*0.98,
+        trunk           = walking_speed*0.96,
```

Comment:
Would it be possible to generalize the walking profiles, you have 3 files to change here, they may end up not being identical! You can open an issue to track this.

Reply:
https://github.com/chairemobilite/transition/issues/864